README.md

```diff
@@ -13,16 +13,24 @@ after a set time period discourages users from using the server as a permanent f
 * yarn (https://yarnpkg.com/ - for the kiwiirc plugin UI)
 * dep (https://github.com/golang/dep - for the server)
 
+
+#### Downloading the file upload server
```

Comment:
This section's heading could be misread as if it was for downloading a compiled release. Maybe "Downloading the file upload server sources"?